src/applications/static-pages/vre-chapter36/Chapter36CTA.js

```diff
@@ -1,120 +1,26 @@
 import React from 'react';
 
-import { connect } from 'react-redux';
-import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
-import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
-import recordEvent from 'platform/monitoring/record-event';
-import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
-
-import EbenefitsLink from 'platform/site-wide/ebenefits/containers/EbenefitsLink';
 import { rootUrl as chapter36Url } from 'applications/vre/28-8832/manifest.json';
 
 const EDUCATION_CAREER_COUNSELING_PATH = 'education-and-career-counseling';
 
-const Chapter36CTA = props => {
+const Chapter36CTA = () => {
   const isEducationAndCareerPage = window.location.href.includes(
     EDUCATION_CAREER_COUNSELING_PATH,
   );
-  let content;
-  const header = isEducationAndCareerPage ? (
-    <h3 id="follow-these-steps-to-apply-on">
-      Follow these steps to apply online for Chapter 36 services:
-    </h3>
-  ) : (
+  const content = (
     <>
-      <h2>How do I apply?</h2>
-      <h3 id="follow-these-steps-to-apply-fo">
-        Follow these steps to apply online now
-      </h3>
+      {!isEducationAndCareerPage ? <h2>How do I apply?</h2> : null}
+      <a
```

Comment:
Chose not to go with the platform provided CTA widget because users can fill this form out without being authenticated, and that widget presents a login cta by default.